domains/champion.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "itz-champion",
+           "discord": "1174148088080908330",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.SbUjo6KtsF9t9ESHv8nhaD_xh38w98GhOZQcJntPEnU0WN3NAJ6PmfiYuURyn6IaqBf8Ifl3Wnc_aPn_MbG8qtWKdE3xioQTO1Vy_Cy8cgNiRavgBm3jsD4X7HwP6wP9q1WyMcDHbPBO-_jV2JrJPRGZbk7cRjCOjZ0GCTKKdAmEb2krd1QmJy3jLdtVuhcqNMXJrm6QjcWezPJY5MtQf-FSE57dK3HyFIOTMyObVtqLnwpoyWDxP-0W6kj8up6vIcGkS_6_W8YHm_Fd_CRLtyIHeHuUHwhCFV9wjwAD6NIdClz2h7_EoHS-5gujwDqwtrazPRwtBHXo3QU6opDmPg.aqkzsFmYPdIUiVM1jWab_g.6sghnlQ6_2OJWN3LAYcYynnHPCiJebdF9rkdsvPd7MVUgYOoeN09_Tg70Z-NkFOF5hED1xtUUkc9Q6aXgpDznOOBV1N0Oizi5FgXeLX38to.masWzlTotKHNeDF_z08E4Q"
+        },
+    
+        "record": {
+            "MX": ["robloxpacketconfig@gmail.com"]
```

Comment:
This is not how MX records work. email should go in the `owner.email` field

see https://is-a.dev/docs/domain-structure/#record-required